assets/pages/bundles-landing/components/Introduction.jsx

```diff
@@ -3,19 +3,52 @@
 // ----- Imports ----- //
 
 import React from 'react';
+import type { Participations } from 'helpers/abtest';
 
+/* eslint-disable react/no-unused-prop-types */
+type PropTypes = {
+    abTests: Participations,
+}
 
+type HeaderCopy = {
+    line1: string,
+    line2: string,
+}
 // ----- Component ----- //
 
-export default function Introduction() {
+export default function Introduction(props: PropTypes) {
+
+  const headerCopyOptions: {[variant: string]: HeaderCopy} = {
+    control: {
+      line1: 'help us deliver the',
+      line2: `independent journalism the world${String.fromCharCode(160)}needs`,
+    },
+    numberOfSupporters: {
+      line1: 'Over 800,000 readers now help fund the Guardian',
+      line2: 'join them today and support the independent journalism the world needs',
+    },
+  };
 
+  const introParagraphMarkup = (
+    <p className="introduction-text__paragraph">
+      Your support helps to make our journalism possible.
+      We don{String.fromCharCode(39)}t have a billionaire owner pulling the strings and we
+      haven{String.fromCharCode(39)}t put up a paywall. Fund independent, quality journalism
```

Comment:
You should be able to use the HTML character (e.g. `&#39;`) here, as it's inside JSX rather than a string.

E.g.:

```html
We don&#39;t have a billionaire
```